src/fides/api/models/system_history.py

```diff
@@ -45,4 +45,4 @@ def edited_by(self) -> Optional[str]:
         db = Session.object_session(self)
         user: Optional[FidesUser] = FidesUser.get_by(db, field="id", value=self.user_id)
 
-        return user.username if user else None
+        return user.username if user else self.user_id
```

Comment:
i see this called out as an additional change. just checking, this isn't specifically related to the system-level cookie support, right? that's OK if it's not, just wanted to make sure i'm not misunderstanding/missing any implications

Reply:
Could have done better at motivating why here. The whole reason I looked into doing this PR in the first place was to be able to make use of  the system history captured when "update_system" was called.  We are going to be having "compass" update the system, not a user.  This will keep "None" from showing up in the FE as the editing user.

Reply:
ah got it, thanks for that context! makes sense 👍